packages/lodestar/test/utils/mocks/chain/chain.ts

```diff
@@ -51,6 +51,13 @@ export class MockBeaconChain extends EventEmitter implements IBeaconChain {
     return this.epochCtx.copy();
   }
 
+  public async getUnfinalizedBlocksAtSlots(slots: Slot[]): Promise<SignedBeaconBlock[]|null> {
+    if (!slots) {
+      return [];
+    }
+    return await Promise.all(slots.map(this.getBlockAtSlot));
```

Comment:
We could rename `getBlockAtSlot` to `getCannonicalBlockAtSlot` and keet this method `getBlocksAtSlot`?

Reply:
> We could rename getBlockAtSlot to `getCannonicalBlockAtSlot`

this is good to me

> and keet this method getBlocksAtSlot

as this method only gets unfinalized blocks, should we keep it `getUnfinalizedBlocksAtSlots` to avoid any confusion? Or any better name?

Reply:
> as this method only gets unfinalized blocks, should we keep it getUnfinalizedBlocksAtSlots to avoid any confusion? Or any better name?

oh scratch that, thought this method is on forkchoice.